package.json

```diff
@@ -134,6 +135,7 @@
     "shelljs": "^0.5.3",
     "sinon": "1.10.2",
     "uglify-js": "^2.5.0",
+    "videojs-contrib-eme": "http://github.com/gesinger/videojs-contrib-eme#options-rework",
```

Comment:
Reminder that this needs to point to a version of contrib-eme with necessary changes.